tests/integration/common/checks-after-conversion/test_enabled_repositories.py

```diff
@@ -3,47 +3,44 @@
 
 import pytest
 
-
-def _check_enabled_repos_rhel8(enabled_repos):
-    """Helper function to assert RHEL repositories."""
-    baseos_repo = "rhel-8-for-x86_64-baseos-rpms"
-    appstream_repo = "rhel-8-for-x86_64-appstream-rpms"
-
-    assert baseos_repo in enabled_repos
-    assert appstream_repo in enabled_repos
+from conftest import SystemInformationRelease
 
 
-def _check_eus_enabled_repos_rhel8(enabled_repos):
-    """Helper function to assert EUS repositories."""
-    baseos_repo = "rhel-8-for-x86_64-baseos-eus-rpms"
-    appstream_repo = "rhel-8-for-x86_64-appstream-eus-rpms"
+def _check_enabled_repos_rhel8(enabled_repos: str = None, eus: bool = False):
```

Comment:
If someone forget to pass the enabled_repos by mistake, this assert would fail with: 

```python
>>> "a" in None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'NoneType' is not iterable
```

```suggestion
def _check_enabled_repos_rhel8(enabled_repos: str = "", eus: bool = False):
```

If you want to be somewhat defensive, you could just change it to be an empty string instead of None. This won't fail with an exception, but will fail with your expected assertion.

Reply:
Yeah, you are right, thanks for that! I'm going to add it